utils/check_quickstart_uniqueness.ts

```diff
@@ -67,4 +61,4 @@ if (require.main === module) {
   main();
 }
 
-module.exports = main;
+module.exports = main;
```

Comment:
Let's update this export now that we have typescript!
We can totally remove this line
```suggestion
```
And update the `const main = () =>` function to be `export default const main = () =>`